hr_applicant/models/hr_app_rel_edu.py

```diff
@@ -0,0 +1,138 @@
+from odoo import api, fields, models
+from odoo.tools.translate import _
+
+
+class ApplicantRelative(models.Model):
+
+    _name = "applicant.relative"
+    _description = "Applicant Relatives"
+    _rec_name = "name"
```

Comment:
When you have name field in your columns, you don't need to define that field in _rec_name.